mat64/cholesky.go

```diff
@@ -178,3 +178,155 @@ func (s *SymDense) InverseCholesky(chol *Cholesky) error {
 	s.SymOuterK(1, &t)
 	return err
 }
+
+// RankOne performs a rank-1 update (or downdate when alpha is negative) of the
```

Comment:
Remove parenthetic; both are updates and "downdate" is not a word outside urbandictionary.